test/unit/spec/appWidgets/validationSpec.js

```diff
@@ -85,324 +109,340 @@ define([
             jasmine.Ajax.uninstall();
         });
 
-        it('validateWorkspaceObjectName - returns valid when they should not exist', (done) => {
-            Validation.validateWorkspaceObjectName(wsObjName, {
-                shouldNotExist: true,
-                workspaceId: 1,
-                workspaceServiceUrl: fakeWsUrl,
-                types: [wsObjType],
-            }).then((result) => {
-                expect(result).toEqual({
-                    isValid: true,
-                    messageId: undefined,
-                    errorMessage: undefined,
-                    shortMessage: undefined,
-                    diagnosis: Constants.DIAGNOSIS.VALID,
-                    value: wsObjName,
-                    parsedValue: wsObjName,
+        describe('validateWorkspaceObjectName', () => {
```

Comment:
added some `describe` blocks around these tests to separate them out and make the results easier to read